### DISCUSSION TITLE
fieldmap assignment

### DISCUSSION THREAD

**Discussion by LancyCao0223** [HALFpipe/HALFpipe#645] [Q&A]
hello, 

I tried multiple ways of assigning fieldmaps to every run but none could be detected or used for any image. As the screenshot shows: in the first column, I have 8 runs for task "cue", 8 runs for task "prob". Following the first column, I have 20 field map columns for subjects 245 to 255, each subject*** has a [Field map run "sub-***"] and [Field map run "sub-***_e2_ph"]. 

I don't know how should i properly assign the fieldmaps so that every subject and every run has a fieldmap. 

Thank you so much for your help. Please let me know if anything is unclear. 

Best,
Lancy
<img width="1230" alt="Screenshot 2023-11-08 at 9 59 37 AM" src="https://github.com/HALFpipe/HALFpipe/assets/150283570/b631bc01-eaf5-44db-ba4f-ae6dbf388ac0">
<img width="1202" alt="Screenshot 2023-11-08 at 10 01 38 AM" src="https://github.com/HALFpipe/HALFpipe/assets/150283570/1dba6d37-4ba7-4c3c-99df-57746bec0fb5">

**Comment by HippocampusGirl** [comment]
Hi @LancyCao0223,
I think you may have found a bug. Could you send me a screenshot of your directory layout for the data?

Best,
Lea

**Reply by LancyCao0223** [reply]
Hi Lea, 
Thank you for your quick response! 
<img width="547" alt="Screenshot 2023-11-08 at 1 39 37 PM" src="https://github.com/HALFpipe/HALFpipe/assets/150283570/e2a646de-e9a1-4471-a1b4-d61c3bc73570">

**Comment by LancyCao0223** [comment]
If we convert data to bids in ahead, we can bypass field map assignments.
